lib/components/full_page_error_indicator.dart

```diff
@@ -1,19 +1,27 @@
 import 'package:flutter/material.dart';
+import 'package:seeds/components/flat_button_long.dart';
 import 'package:seeds/constants/app_colors.dart';
 import 'package:seeds/i18n/components/components.i18n.dart';
 
 class FullPageErrorIndicator extends StatelessWidget {
   final String? errorMessage;
+  final String? buttonTitle;
+  final Function()? buttonOnPressed;
```

Comment:
we call these "onButtonPressed" - please rename so it's the same as everywhere else in the app.

Reply:
Also I suggest to prefer use VoidCallback typedef.

Reply:
> we call these "onButtonPressed" - please rename so it's the same as everywhere else in the app.

Done!

Reply:
> Also I suggest to prefer use VoidCallback typedef.

Done!